Javascript/100-Problems/script.js

```diff
@@ -850,3 +850,472 @@ const studentWithLowestPercentageInSubject = (subjectName) => {
 };
 
 console.log(studentWithLowestPercentageInSubject("Maths"));
+
+//function to find and print the subject(s) with the highest/lowest percentage of marks for a specific student
+
+const printSubjectPercentageOfStudent = (
+  studentName,
+  option,
+  obtainedPercentage
+) => {
+  const students = classObj.students;
+  const totalmark = 50;
+  let subjectName = "";
+  let subjectArray = [];
+  let subjectMark = 0;
+
+  students.find((student) => {
+    const subjects = student.marks;
+
+    if (student.name === studentName) {
```

Comment:
What if there is two guys with same names. Always use a unique property to find a specific value like id. We have mentioned it before right?.